ospd/ospd.py

```diff
@@ -978,6 +1007,62 @@ def handle_get_vts_command(self, vt_et):
 
         return simple_response_str('get_vts', 200, 'OK', responses)
 
+    def handle_get_performance(self, scan_et):
+        """ Handles <get_performance> command.
+
+        @return: Response string for <get_performance> command.
+        """
+        start = scan_et.attrib.get('start')
+        end = scan_et.attrib.get('end')
+        titles = scan_et.attrib.get('titles')
+
+        cmd = list()
+        cmd.append('gvmcg')
```

Comment:
Can be sumarized in one line
```suggestion
        cmd = ['gvmcg']
```